keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1332,8 +1335,16 @@ with “volume” replaced by “resource” and volume specific parts removed.
 This RPC is called by kubelet when a Pod that wants to use the
 specified resource is scheduled on a node.  The Plugin SHALL assume
 that this RPC will be executed on the node where the resource will be
-used.  The Plugin SHALL return device name and kind for allocated
-device[s].
+used.  ResourceClaim.meta.Namespace, ResourceClaim.meta.UID,
+ResourceClaim.Name, ResourceClaim.meta.Namespace and
```

Comment:
@bart0sh @pohly Sorry for seeing it after it's already merged, but there is "ResourceClaim.meta.Namespace" two times.